build.gradle

```diff
@@ -89,6 +97,8 @@ dependencies {
     modRuntimeOnly "com.terraformersmc:modmenu:${project.mod_menu_version}"
     modImplementation "maven.modrinth:sodium:${project.sodium_version}"
     modCompileOnly "de.florianmichael:ViaFabricPlus:${project.viafabricplus_version}"
+    modImplementation "meteordevelopment:baritone:${project.baritone_version}-SNAPSHOT"
+    implementation 'dev.babbaj:nether-pathfinder:1.5'
```

Comment:
I should point out that these libraries will not be included in the release JAR. Using Meteor's Baritone is fine, as I think most people use Meteor's Baritone and that could eliminate conflicts thanks to Fabric's mod system, but they don't really provide downloads for version 1.21.1, and neither do we.

Luckily we could integrate the Baritone Mod as toggleable mod via the LiquidLanucher, since it can pull mods from any Maven repository - but I might consider making an upstream of their maven or deploy our own, as well as provide a better site for downloading baritone, because for some reason there is no official site and GitHub releases of the official are not being maintained.

Reply:
Sure. The official baritone doesn't work with 1.21.1 without conflicts, so I used MeteorClient baritone. Officially, baritone supports 1.21.2, so we can drop MeteorClient baritone when LiquidBounce will support 1.21.2 and above.